### ISSUE TITLE
Edit question either leads to 'robot of undefined' bug or the updated answer is not reflected in the UI. 

### ISSUE THREAD

**Issue by sbalajime** [space10-community/conversational-form#437]
Code sample - https://stackblitz.com/edit/react-conversational-form-example-jheej2?file=myForm.js
Bug Description:
The above code sample has conditional branching. After entering height and weight, if i click on the height answer to edit it, It is not updated in the UI . 

Version - 1.0.2

Browser - Chrome 90.0.4430.212

**Comment by jhadraoui** [comment]
Did you find a solution to this problem ? 
I have similar issues, when trying to go back and edit a previous answer, the whole flow is affected and the questions and answers get all mixed up

**Comment by sbalajime** [comment]
@jihanehk95 No. I have faced so many issues with this package. So i am using tripetto for conversational forms.
